model/config_test.go

```diff
@@ -1326,3 +1326,21 @@ func TestConfigMarketplaceDefaults(t *testing.T) {
 		require.Equal(t, "https://marketplace.example.com", *c.PluginSettings.MarketplaceUrl)
 	})
 }
+
+func TestSetDefaultFeatureFlagBehavour(t *testing.T) {
```

Comment:
to fix name.

Reply:
Sorry I don't understand this comment.

Reply:
My bad, wasn't aware that behavour is a word, looks like it is the current behavior is an alteration of that.

Reply:
Oh, I see the typo now:
```suggestion
 func TestSetDefaultFeatureFlagBehaviour(t *testing.T) { 
```